.golangci.yml

```diff
@@ -6,93 +6,34 @@ run:
   go: '1.19'
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
```

Comment:
`enable-all` option is Un-deprecated. 
https://github.com/golangci/golangci-lint/issues/1888